codecov.yml

```diff
@@ -0,0 +1,58 @@
+coverage:
+  range: 60..100
+  round: down
+  precision: 1
+  status:
+    project:
+      default:
+        # Avoid false negatives
+        threshold: 1%
+
+comment:
+  behavior: default
+  layout: "condensed_header, condensed_files, condensed_footer" # add "condensed_" to "header", "files" and "footer"
+  hide_project_coverage: false # set to true
+  require_changes: true  # if true: only post the comment if coverage changes
+  require_base: false        # [true :: must have a base report to post]
+  require_head: true       # [true :: must have a head report to post]
+
+
+codecov:
+  require_ci_to_pass: false
+# ref: https://docs.codecov.com/docs/codecovyml-reference
+
+ignore:
+  - "protocol"
```

Comment:
I think sorted/grouped files would be easier to keep track of and quickly understand what files are ignored within a directory